scripts/lint

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -eu
+
+flake8 . --exclude=./improved-funicular/ --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
I had to `pip install flake8==3.7.9` -- you want to add that to requirements.txt?

Reply:
Ah good catch!